pkg/server/applications.go

```diff
@@ -0,0 +1,777 @@
+package server
```

Comment:
This diff is a little problematic. This will blow away any git history on the original `server.go` file. Is there a way we can get `git` to recognize this as a file rename instead of a new file?